src/haz3lweb/view/CtxInspector.re

```diff
@@ -3,21 +3,27 @@ open Node;
 open Util.Web;
 
 let context_entry_view =
-    ((name: string, {typ, _}: Haz3lcore.Ctx.entry)): Node.t =>
+    (
+      (
+        name: string,
+        {item: typ, _}: Haz3lcore.TermCtx.entry(Haz3lcore.Typ.t),
+      ),
+    )
+    : Node.t =>
   div(
     ~attr=clss(["context-entry"]),
     [text(name), text(":"), Type.view(typ)],
   );
 
 let ctxc = "context-entries";
 
-let exp_ctx_view = (ctx: Haz3lcore.Ctx.t): Node.t =>
+let exp_ctx_view = (ctx: Haz3lcore.TermCtx.t): Node.t =>
```

Comment:
still concerned about not having a single context if we want to view a context in order, including type variables / aliases that are in scope. remind me again why it's beneficial to maintain them separately?

Reply:
1. It's easier to debug and maintain because one lookup results in only one sort of term.
2. It separates the domain of expression variable names from the type variable names, which makes things easier to track.
3. (newly mentioned) It might not be a bad idea to display two different sorts of contexts separately.

I understand it's concerning when we lose information. If the reasons above don't seem to be convincing, I've drafted a mixed context and we can switch to that too.